src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-compliance.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Compliance mode
+description: Learn how to install and configure the Nuxeo Retention addon in the Compliance mode.
+review:
+    comment: ''
+    date: '2019-08-05'
+    status: 'ok'
+labels:
+    - lts2019-wip
+    - grenard
+    - jaubenque
+    - retention-management
+toc: true
+tree_item_index: 950
+---
+
+This page gives all the necessary steps to install the Retention Management addon with the [Compliance mode]({{page page='nuxeo-retention-management'}}#configuration-modes).
+
+## Prerequisites
+
+The [Amazon S3 Online Storage]({{page version='' space='nxdoc' page='amazon-s3-online-storage'}}) addon needs to be configured on your instance
+
+## Installation
+
+{{{multiexcerpt 'MP-installation-easy' space='nxdoc' page='Generic Multi-Excerpts'}}}
+
+## Configuration
+
+### Nuxeo Server
+
+In Compliance mode, the Nuxeo Retention Management addon requires to disable the **attachments**, **versioning**, and **comments** features.
+
+This is done adding the parameter `nuxeo.retention.compliance.enabled=true` in the server [nuxeo.conf]({{page page='configuration-parameters-index-nuxeoconf'}}).
+
+This has to be done prior to any usage of the Nuxeo instance to guarantee the data consistency.
+
+### Amazon S3
+
+In compliance mode, Nuxeo Platform with Nuxeo Retention Management addon requires the usage of 2 Amazon buckets:
+
+- A standard S3 bucket as for any other standard Nuxeo instance: this bucket is used to store the standard documents. You can refer to [Amazon S3 Online Storage]({{page version='' space='nxdoc' page='amazon-s3-online-storage'}}) documentation to configure this bucket.
+
+- An S3 bucket dedicated to the records that is configured on [Compliance mode](#s3-configuration-requirements): this bucket is used to store the records only. Compared to the standard bucket, the Garbage Collector must be disabled as there is a [specific deletion process]({{page page='nuxeo-retention-technical-overview'}}#record-deletion-flow).
+
+#### {{> anchor 's3-configuration-requirements'}} Amazon S3 Configuration Requirements
+
+Compliance with [SEC Rule 17a-4]({{page page='nuxeo-retention-management'}}#sec-rule-17a4) requires to configure Nuxeo Platform to use Amazon S3 Compliance Buckets to store records required by regulation, thereby establishing the foundation for meeting the requirements of the Rule.</br>
+
+Specifically:
+
+- Direct writes to the Amazon S3 storage system must be disabled, to ensure all documents transit through Nuxeo Platform for compliant processing.
+
+- The Amazon S3 Object Lock feature must be enabled in Compliance Mode on the bucket intended to store objects that are marked as final records.
```

Comment:
`The Amazon S3 Object Lock feature must be enabled` is enough. Nuxeo app server will set the binary under retention with the proper mode (compliant or governance)

Reply:
I know it will be done automatically by Nuxeo, but as it is a mandatory condition for the compliance, I prefer to keep it